entries/u/ucalgary.ca.json

```
@@ -0,0 +1,20 @@
+{
+  "University of Calgary": {
+    "domain": "ucalgary.ca",
+    "tfa": [
+      "custom-software",
```

Microsoft's auth app is advertised but any TOTP app works.
```suggestion
      "custom-software",
      "totp",
```